### PR TITLE
Add ReSharper gallery badge.

### PR DESCRIPTION
**Pull request by matkoch**
Also changed the description from 'JetBrains Plugins' to 'JetBrains IntelliJ IDEA Plugins'. Otherwise, it would be ambiguous.

### CODE REVIEW

File: lib/all-badge-examples.js [hunk 1]
```diff
@@ -1114,9 +1114,13 @@ const allBadgeExamples = [
         previewUri: '/jenkins/plugin/v/blueocean.svg'
       },
       {
-        title: 'JetBrains Plugins',
+        title: 'JetBrains IntelliJ IDEA Plugins',
         previewUri: '/jetbrains/plugin/v/9630-a8translate.svg'
       },
+      {
+        title: 'JetBrains ReSharper Plugins',
+        previewUri: '/resharper/v/ReSharper.Nuke.svg'
```
Comment (chris48s) [inline lib/all-badge-examples.js]: Can you also add an example for the `/vpre` endpoint here and an example under the downloads section for the `/dt` endpoint you've added.

**Comment by matkoch** [comment]
Done.

**Comment by matkoch** [comment]
Well, PyCharm, WebStorm and others are all based on IntelliJ. “JetBrains plugins” is  definitely more “wrong”. Because ReSharper is also a product and plugins from plugins.jetbrains.com don’t apply to it.

But it’s

**Comment by platan** [comment]
@matkoch good point, merged!

**Comment by matkoch** [comment]
Thanks! :) When will this be live?

**Comment by platan** [comment]
@matkoch Unfortunately I cannot give you any date. Thaddée [@espadrine], who has limited time on this project, is the only sysadmin. I the past deploys usually happened every 1–3 weeks, but this changed recently and last deploy was on March 25.

**Pull request merged**